templates/vue/src/components/Lightbox/media/MultiContentMedia/MultiContentRows.vue

```diff
@@ -104,12 +104,6 @@ export default {
     complete(rowId) {
       this.$emit("complete", rowId)
     },
-    changeRow(rowInfo) {
-      this.$emit("change-row", { context: this.context, ...rowInfo })
-    },
-    areAllPopup(nodes) {
-      return nodes.every(node => node.popup !== null)
-    },

```

Comment:
Can you explain why these are removed, and same in the index file too?

Reply:
I did a global search for these methods in our code base, and they are only defined but never called. Since they are unused, I just removed them to avoid confusion in the future.